src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', e => {
+  const list = e.target.firstElementChild;
+
+  if (list.tagName === 'UL') {
```

Comment:
Check if the 'list' variable exists before accessing its properties to avoid potential errors.